src/platform/packages/shared/kbn-alerts-as-data-utils/src/schemas/create_schema_from_field_map.ts

```diff
@@ -211,6 +211,8 @@ const generateSchemaLines = ({
           lineWriter.addLine(`${keyToWrite}: ${getSchemaDefinition('schemaUnknown', isArray)},`);
         }
         break;
+      case 'unmapped':
+        break;
```

Comment:
We keep the unmapped fields only in the filed_map but don't reflect it to index mapping.